### ISSUE TITLE
Feature request: sortable table rows

### ISSUE THREAD

**Issue by ohbendy** [ohbendy/Scripts-of-SEAsia#3]
Some people will want to sort by country, language, perhaps Unicode status.

**Comment by ohbendy** [comment]
Thanks for looking at this! I did wonder what would happen with the empty spaces. I'm not sure how to deal with those, it's going to add extra clutter if every script name is repeated for every language, and if we also have to add an 'orthography name' to every language, to be able to sort by orthography, we'll have to invent names for the orthographies that don't really have names. Thoughts?

**Comment by sovichet** [comment]
I think this issue has been solved already. Closing now.

**Comment by ohbendy** [comment]
I noticed we've got the same problem of empty cells when filtering. Here we see Tai Sar is listed under Round Script (reformed) but when we filter and the Shan row is not included, that category disappears. Will that be easier to manage with a .js source instead of .md? 

![Screenshot 2024-07-28 at 18 55 32](https://github.com/user-attachments/assets/3c202317-cd51-40de-9dde-e25d6f4c36d7)

![Screenshot 2024-07-28 at 18 52 44](https://github.com/user-attachments/assets/2905f7bd-ca9e-47a5-a3f1-8d3d6f5166b2)

**Comment by sovichet** [comment]
This is from the v2 codebase search/filter. Can you confirm this is correct?

![Screenshot 2024-08-20 at 5 36 30 in the afternoon](https://github.com/user-attachments/assets/5064ce29-45f0-478c-afed-f37556c3d15a)

**Comment by ohbendy** [comment]
Sorry I see I didn't explain properly. Here's the problem:

Tai Sar appears with various orthographies in the full list before searching:

![tai sar yes](https://github.com/user-attachments/assets/49ec97ec-d6c9-4079-9dc2-543f1601a01a)

But when searching for it, the orthographies disappear because they're on lines that aren't displayed, but we need those to show:

![Tai Sar no](https://github.com/user-attachments/assets/8aa8741b-10ac-49e9-a451-20ac60d1608a)

**Comment by sovichet** [comment]
Ah I see. Technically, in the current data/table, there isn't `Long script (traditional) (Lik Hto Ngouk, Lik To Yao)`, etc. in those rows, so it's not showing. I'd suggest repeating the text for those rows.

Currently, we are grouping the rows by script; if we are to also group them by Orthography and it's happening in all other rows, we can consider updating the code.

**Comment by ohbendy** [comment]
Right, but also the script isnt repeated in those rows, but it shows when searching. Can't it do the same for orthographies? I don't want to fill up the table with repeated info

**Comment by sovichet** [comment]
I guess we had to go with the repeated info for this static data. I think it is already faster and easier to implement and display the data this way.
